src/plugins/codeStyle/index.spec.ts

```diff
@@ -828,6 +828,211 @@ describe('codeStyle', () => {
             expect(actualSrc).to.equal(expectedSrc);
         });
 
+        // it('BRS file color format is quotedNumericHex and case is uppercase', async () => {
```

Comment:
When will/should these tests be reenabled? if never, can you delete the commented-out code?